drivers/gpio/Kconfig.qmsi

```diff
@@ -50,12 +50,12 @@ config GPIO_QMSI_0
 
 config GPIO_QMSI_0_NAME
 	string "Driver name"
-	depends on GPIO_QMSI_0
+	depends on GPIO_QMSI_0 && !HAS_DTS
```

Comment:
Are there any cases for !HAS_DTS?  Any reason to keep these Kconfig symbols?

Reply:
Has it's been done in all other Kconfig places, config options there are kept with that dependency. 
Let's remove all these options when we are done porting everything to dts, and sure it works properly.

Reply:
> Has it's been done in all other Kconfig places, config options there are kept with that dependency.
> Let's remove all these options when we are done porting everything to dts, and sure it works properly.

Its been the case that we've been removing Kconfig options if there aren't any uses for non-DTS.  So if its the case that its all users of GPIO_QMSI_0_NAME have HAS_DTS=y, than we should remove the Kconfig symbol.